### ISSUE TITLE
How can we change shadowGenerator darkness value?

### ISSUE THREAD

**Issue by snippet-nicotine** [brianzinn/react-babylonjs#36]
Hello,

I'm new on react-babyonjs and it feels very promising! But starting simple on changing the value of the shadow's generator darkness semme's not working.

```JSX
<Scene>
                <arcRotateCamera name="arc" target={new Vector3(0, 1, 0)}
                                 alpha={-Math.PI / 2} beta={(0.5 + (Math.PI / 4))}
                                 radius={4} minZ={0.001} wheelPrecision={50}
                                 lowerRadiusLimit={8} upperRadiusLimit={20} upperBetaLimit={Math.PI / 2}
                                 inertia={0.5}
                                 inertialPanningX={0} inertialPanningY={0}
                                 speed={10}
                />
                <hemisphericLight name='hemi' direction={new Vector3(0, -1, 0)} intensity={0.8}/>
                <directionalLight name="shadow-light" setDirectionToTarget={[Vector3.Zero()]} direction={Vector3.Zero()}
                                  position={new Vector3(-40, 30, -40)}
                                  intensity={1} shadowMinZ={1} shadowMaxZ={2500}>

                    <shadowGenerator mapSize={1024} useBlurExponentialShadowMap={true} blurKernel={32}
                                     shadowCasters={["sphere1"]} forceBackFacesOnly={true} depthScale={100}
                                     blurScale={1} setDarkness={0.5}
                    />

                </directionalLight>
                <sphere name="sphere1" diameter={2} segments={16} position={new Vector3(0, 2.5, 0)}>
                </sphere>

                <ground name="ground1" width={10} height={10} subdivisions={2} receiveShadows={true}>
                </ground>
            </Scene>
```

How i'm supposed to do in order to change the darkness value?
Is there something I missed?

Thank you

**Comment by brianzinn** [comment]
I think you just need to pass as an array. I’ll update the code to pass through primitive data types as a single argument.

**Comment by brianzinn** [comment]
Sorry it took so long :)  I finally had time to sit down and figure it out!  It was a combination of two things.  First I needed to have the single parameter methods handled correctly.  Secondly, the `ShadowGenerator` is created differently than regular BabylonJS objects, because it needs a light passed in the constructor.  So, the `ShadowGenerator` creation is delayed as it needs to wait for the light to be created and was losing therefore the initial non-constructor props.  I should get that pushed in next couple of days and will msg you here when there is a new NPM.  You can see if referenced commit the diff. ☘️

**Comment by brianzinn** [comment]
You can use `setDarkness={0.5}` and on newer versions of babylonjs `darkness={0.5}`.  Published NPM `2.0.6`.
[storybook example](https://brianzinn.github.io/react-babylonjs/?path=/story/physics-and-hooks--bouncy-playground)
